### DISCUSSION TITLE
Updating old software to latest version of osgearth

### DISCUSSION THREAD

**Discussion by paul-jewell** [pelicanmapping/osgearth#2111] [Support Q&A]
Good morning!
I am updating some software which uses osgearth, and have come across a function which doesn't appear in the current version: 
`skyNode->setMinimumAmbient(osg::Vec4(d, d, d, 1.0f)); `
In the current version, I have searched through the include files and documentation, but I can't see what I should update this with. Could you please point me at the correct function?  I have 3.2 installed, but will move to the 3.3 version as soon as I can get past the current challenges!
Many thanks!

**Comment by paul-jewell** [comment]
Looks like this is the correct replacement:
` skyNode->getSunLight()->setAmbient(osg::Vec4(d, d, d, 1.0f));`
Although I am working through some other changes so can't yet confirm that I have the same functionality!
I would still welcome your comments as to whether this is correct, or incorrect!
Thanks.

**Reply by gwaldron** [reply]
I believe that's right. (Sorry for the delayed response, I didn't see this pop up on my feed!)
